src/views/Players/UpdatePlayer.test.jsx

```diff
@@ -0,0 +1,76 @@
+import {
+  render,
+  screen,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { setupServer } from 'msw/node';
+import { rest } from 'msw';
+import { createMemoryHistory } from 'history';
+import { Router, Route } from 'react-router-dom';
+import UpdatePlayer from './UpdatePlayer.jsx';
+import PlayerList from './PlayerList.jsx';
+
+const mockPlayer = {
+  id: 13,
+  created_at: '2021-12-10T01:29:22.974Z',
+  name: 'test name',
+  position: 'umpire',
+  team_id: 1,
+  teams: {
+    name: 'something',
+  },
+};
+
+const server = setupServer(
+  rest.get(
+    `${process.env.REACT_APP_SUPABASE_URL}/rest/v1/players`,
+    (req, res, ctx) => {
+      return res(ctx.json([mockPlayer]));
+    }
+  ),
+  rest.patch(
+    `${process.env.REACT_APP_SUPABASE_URL}/rest/v1/players`,
+    (req, res, ctx) => {
+      return res(ctx.json(mockPlayer));
+    }
+  ),
+  rest.get(
```

Comment:
the won't ever get called because the first `rest.get` will always win. this presented quite a challenge for lots of folks, and the solution that we figured out was to edit the actual values of the `mockPlayer` object within your test (see below)

Reply:
I misread the code here! the solution still works, but I see now that this second `get` was intercepting the `/v1/teams` call